faillint/faillint.go

```diff
@@ -201,3 +211,50 @@ func isTopName(n ast.Expr, name string) bool {
 	id, ok := n.(*ast.Ident)
 	return ok && id.Name == name && id.Obj == nil
 }
+
+func parseDirective(s string) (option string) {
+	if !strings.HasPrefix(s, "//faillint:") {
+		return ""
+	}
+	s = strings.TrimPrefix(s, "//faillint:")
+	fields := strings.Split(s, " ")
+	return fields[0]
+}
+
+const (
+	ignore     = "ignore"
+	fileignore = "file-ignore"
+)
```

Comment:
Can you move them to the top of the file? Also I think documenting them and naming them in a more descriptive way would be better. Such as: `ignoreKeyE` and `fileIgnoreKey`

Reply:
Refactored now; in 08c105025674e61b994c3cabe065f660728128c2.